keras_cv/layers/preprocessing/README.md

```diff
@@ -0,0 +1,48 @@
+# Base Layers
+
+- BaseImageAugmentationLayer
+- VectorizedBaseImageAugmentationLayer
+- RandomAugmentationPipeline
+
+# Preprocessing Layers
```

Comment:
Let's add a little text about how the augmentation API can be used, what the data i/o format is like, and what the meaning of this table is.